pkg/controller/render/render_controller.go

```diff
@@ -427,6 +427,7 @@ func (ctrl *Controller) syncGeneratedMachineConfig(pool *mcfgv1.MachineConfigPoo
 	_, err = ctrl.mcLister.Get(generated.Name)
 	if apierrors.IsNotFound(err) {
 		_, err = ctrl.client.MachineconfigurationV1().MachineConfigs().Create(generated)
+		glog.Infof("Generated machineconfig %s from %d sources", generated.Name, len(configs))
```

Comment:
I like this. How about we also print the names of all the source configs?

Reply:
already logs here
https://github.com/openshift/machine-config-operator/pull/253/files#diff-9d1c255b5e10f6159f19c5fafa976458R413

Reply:
That one seems to happen though even if nothing changed; gets verbose.  I deleted it.